lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -42,14 +42,12 @@ public function __construct($directory, $extension = self::EXTENSION)
      */
     protected function doFetch($id)
     {
-        $filename = $this->getFilename($id);
+        $value = @include $this->getFilename($id);
```

Comment:
yes, yes, I know there's an error suppression here.

We can't do otherwise, because this is the only way to avoid `is_file` and `is_readable` calls.